### ISSUE TITLE
add term for salinity categories

### ISSUE THREAD

**Issue by wdduncan** [GenomicsStandardsConsortium/mixs#119]
For the NMDC, we are finding a number of submissions that refer to categories of salinity rather than a measurement value (i.e., a number and a unit). Examples of salinity categories:
- Halophile
- Euryhaline
- Stenohaline
- Halotolerant

There is some info on the [Wiki page[(https://en.wikipedia.org/wiki/Salinity#Classification_of_water_bodies_based_upon_salinity) too.

cc @cmungall

**Comment by ramonawalls** [comment]
I agree it is import to make this a separate field. I suggest we pull a list of values from ENVO, or request new terms as needed.

**Comment by ramonawalls** [comment]
Be sure new terms is added to all the packages where salinity us used.

**Comment by cmungall** [comment]
These are the distinct values in INSDC that don't start with a number

Missing: Not provided
not provided
NA
missing
not collected
not applicable
Missing:Not reported
NOT APPLICABLE
Missing: Not Provided
N/A
NULL
Halophile
Not collected
-999
Unknown
Not applicable
Missing: Not collected
fresh water
FW
Halotolerant
nd
&gt;50%
Stenohaline
{0.0} {%}
&lt; 0.1 ppt
salt water
not recorded
unknown
{19.4} {%}
{16.6} {%}
{19.1} {%}
salt Required
xx
{1.03} {%}
Original
NaCl 0 - 3% (w/v)
{0.90} {%}
salt 240-300 g/L
salt 100-400 g/L
{4.76} {%}
{0.02} {%}

**Comment by only1chunts** [comment]
Some thoughts on this topic from me.
most of the terms listed above (@cmungall's comment) are pretty meaningless for the current field of salinity and only a couple of them would be suitable in the new field of salinity category,e.g. Stenohaline, fresh water, and sea water.

Halophile and halotolerant are terms to describe the organism rather than the water, so probably not suitable here.

Is the new term actually a more specific way to ask for the fine-grain environmental descriptor material/medium i.e. the expected values would also fit in the MIXS term "environmental medium" MIXS:0000014

I like the idea of making it simpler for users to understand which values to include, but by splitting the ENVO terms into environment-specific terms like this are we making it more difficult to amalgamate metadata later? 
Is this actually another example of how, if we provide the checklist specific Slims (#118) for the 3 ENVO fields it will alleviate the problem and remove the requirement for this new term?

Possible values for the slim of "Environmental medium" [MIXS:0000014] valid values:
Fresh water [ENVO:00002011]
Saline water [ENVO:00002010]
Brackish water [ENVO:00002019]
Brine [ENVO:00003044]
moderately saline water [ENVO:01001038]
slightly saline water [ENVO:01001037]
Euryhaline or euhaline (not yet in ENVO)
Stenohaline (not yet in ENVO)
hyperhaline (not yet in ENVO)
metahaline (not yet in ENVO)
mixoeuhaline (not yet in ENVO)
polyhaline (not yet in ENVO)
mesohaline (not yet in ENVO)
oligohaline (not yet in ENVO)
 NB - All the above terms not in ENVO are from the Wikipedia page Bill provided above
